.gitignore

```diff
@@ -21,8 +21,6 @@ vendor
 
 # Compiled docs.js
 .cache/
-resources/web/docs.js
```

Comment:
Instead of writing this here I write it directly to the destination when building all docs or to `/tmp/parcel` when building a single book.